docs/Inputs.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "Input Components"
```

Comment:
please revert the formatting changes in this file, and disable prettier for Markdown in your config. This type of change is a pain to review, may break stuff, and brings no value. Our hand crafted doc is better than what prettier does.